autoscale/src/test/java/com/sequenceiq/periscope/endpointtests/DistroXAutoScaleScalingActivityV1EndpointTest.java

```diff
@@ -0,0 +1,484 @@
+        ScalingActivity scalingActivity2 = createScalingActivity(testCluster, ActivityStatus.METRICS_COLLECTION_FAILED,
+                Instant.now().minus(100, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_2);
+        ScalingActivity scalingActivity3 = createScalingActivity(testCluster, ActivityStatus.UPSCALE_TRIGGER_FAILED,
+                Instant.now().minus(50, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_3);
+        ScalingActivity scalingActivity4 = createScalingActivity(testCluster, ActivityStatus.UPSCALE_TRIGGER_FAILED,
+                Instant.now().minus(40, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_4);
+        ScalingActivity scalingActivity5 = createScalingActivity(testCluster, ActivityStatus.DOWNSCALE_TRIGGER_SUCCESS,
+                Instant.now().minus(30, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_5);
+        ScalingActivity scalingActivity6 = createScalingActivity(testCluster, ActivityStatus.DOWNSCALE_TRIGGER_FAILED,
+                Instant.now().minus(20, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_6);
+        ScalingActivity scalingActivity7 = createScalingActivity(testCluster, ActivityStatus.ACTIVITY_PENDING,
+                Instant.now().minus(10, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_7);
+        ScalingActivity scalingActivity8 = createScalingActivity(testCluster, ActivityStatus.SCALING_FLOW_IN_PROGRESS,
+                Instant.now().minus(0, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_8);
+
+        scalingActivityRepository.saveAll(Arrays.asList(scalingActivity, scalingActivity2, scalingActivity3, scalingActivity4,
+                scalingActivity5, scalingActivity6, scalingActivity7, scalingActivity8));
+
+        Cluster testCluster2 = new Cluster();
+        testCluster.setStackCrn(TEST_CLUSTER_CRN_2);
+        testCluster.setStackName(TEST_CLUSTER_NAME_2);
+        testCluster.setCloudPlatform("AWS");
+
+        ClusterPertain clusterPertain2 = new ClusterPertain();
+        clusterPertain.setTenant(TEST_ACCOUNT_ID);
+        clusterPertain.setWorkspaceId(TEST_WORKSPACE_ID);
+        clusterPertain.setUserId(TEST_USER_ID.toString());
+        clusterPertain.setUserCrn(TEST_USER_CRN);
+
+        testCluster2.setClusterPertain(
+                clusterPertainRepository.findFirstByUserCrn(clusterPertain.getUserCrn())
+                        .orElseGet(() -> clusterPertainRepository.save(clusterPertain2)));
+        clusterRepository.save(testCluster2);
+
+        ScalingActivity scalingActivity9 = createScalingActivity(testCluster2, ActivityStatus.METRICS_COLLECTION_SUCCESS,
+                Instant.now().minus(150, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_11);
+        ScalingActivity scalingActivity10 = createScalingActivity(testCluster2, ActivityStatus.DOWNSCALE_TRIGGER_SUCCESS,
+                Instant.now().minus(100, MINUTES).toEpochMilli(), TEST_ACTIVITY_CRN_12);
+
+        scalingActivityRepository.saveAll(Arrays.asList(scalingActivity9, scalingActivity10));
```

Comment:
More out of curiosity - what gets wired in here? i.e. where do these values get persisted for the tests.